modules/trustpidSystem.js

```diff
@@ -0,0 +1,222 @@
+ * Properly sets the trustpid acronym depending on the domain value.
+ * @param domain
+ */
+function getDomainAcronym(domain) {
+  let acronym = '';
+  const prefix = '-';
+  switch (domain) {
+    case 'tmi.mno.link':
+      acronym = 'ndye';
+      break;
+    case 'tmi.vodafone.de':
+      acronym = 'pqnx';
+      break;
+    case 'tmi.telekom.de':
+      acronym = 'avgw';
+      break;
+    case 'tmi.tmid.es':
+      acronym = 'kjws';
+      break;
+    case 'uat.mno.link':
+      acronym = 'xxxx';
+      break;
+    case 'es.tmiservice.orange.com':
+      acronym = 'aplw';
+      break;
+    default:
+      return 'none';
+  }
+  return mnoAcronym = prefix + acronym;
+}
+
+// Set a listener to handle the iframe response message.
+window.addEventListener('message', messageHandler, false);
+
+/**
+ * Get the "umid" from html5 local storage to make it available to the UserId module.
+ * @param config
+ * @returns {{trustpid: (*|string), acr: (string)}}
+ */
+function getTrustpidFromStorage() {
+  // Get the domain either from localStorage or global
+  let domain = JSON.parse(window.localStorage.getItem('fcIdConnectDomain')) || mnoDomain;
+  debugModeConsoleLog('info', `getTrustpid domain: ${domain}`);
+
+  if (!domain) {
+    debugModeConsoleLog('info', `getTrustpid domain: no domain`);
+    return {
+      trustpid: null,
+      acr: null,
+    };
+  }
+
+  // Get the acronym from global
+  let acronym = mnoAcronym;
+  // if acronym is empty, but "domain" is available, get the acronym from domain
+  if (!acronym) {
+    getDomainAcronym(domain);
+    acronym = mnoAcronym;
+  }
+
+  // Domain is correct in both local storage and idGraph, but no acronym is existing for the domain
+  if (domain && !acronym) {
+    return {
+      trustpid: null,
+      acr: null
+    }
+  }
+
+  let fcIdConnectObject;
+  let fcIdConnectData = JSON.parse(window.localStorage.getItem('fcIdConnectData'));
+  debugModeConsoleLog('info', `getTrustpid fcIdConnectData:
+  ${JSON.stringify(fcIdConnectData)}`);
+
+  if (fcIdConnectData &&
+    fcIdConnectData.connectId &&
+    Array.isArray(fcIdConnectData.connectId.idGraph) &&
+    fcIdConnectData.connectId.idGraph.length > 0) {
+    fcIdConnectObject = fcIdConnectData.connectId.idGraph.find(item => {
+      return item.domain === domain;
+    });
+  }
+  debugModeConsoleLog('info', `getTrustpid fcIdConnectObject:
+  ${JSON.stringify(fcIdConnectObject)}`);
+
+  return {
+    trustpid: (fcIdConnectObject && fcIdConnectObject.umid)
+      ? fcIdConnectObject.umid
+      : null,
+    acr: acronym,
+  };
+}
+
+/**
+ * Returns whether we are in Prebid debug mode or not.
+ * @returns {boolean}
+ */
+function isDebugMode() {
+  return (window.location.href.indexOf('?pbjs_debug=true') > -1) || (window.location.href.indexOf('&pbjs_debug=true') > -1);
+}
+
+/**
+ * Show debugging messages in browser developer console.
+ * @param type
+ * @param data
+ */
+function debugModeConsoleLog(type = 'info', data) {
+  if (isDebugMode()) {
```

Comment:
you should use `logInfo()` from utils.js instead of this check and directly writing to `console.info()`

Reply:
Done